src/PortingAssistant.Client.Telemetry/Model/APIMetrics.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
```

Comment:
Something to consider since we have added all the telemetry data structure here: in the future we should update [`porting-assistant-dotnet-ui`](https://github.com/aws/porting-assistant-dotnet-ui/tree/develop/packages/csharp/PortingAssistant/PortingAssistant.Telemetry) and [PortingAssistantCLI](https://code.amazon.com/packages/PortingAssistantCLI/trees/mainline/--/PortingAssistantCLI/Model) to use the ones from `PortingAssistant.Client.Telemetry` nuget package. This way we keep our telemetry data consistent, and reduce maintenance overhead. Maybe we should add a ticket/issue to keep track of this.

Reply:
This is AI I already told anrudda, he will work on this once he finished his current work